packages/ckeditor5-dev-changelog/tests/utils/providenewversionformonorepository.test.ts

```diff
@@ -0,0 +1,311 @@
+/**
+ * @license Copyright (c) 2003-2025, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+import { npm } from '@ckeditor/ckeditor5-dev-utils';
+import { beforeEach, describe, expect, it, vi } from 'vitest';
+import inquirer from 'inquirer';
+import semver, { type ReleaseType } from 'semver';
+import {
+	provideNewVersionForMonorepository
+} from '../../src/utils/providenewversionformonorepository.js';
+
+vi.mock( 'inquirer' );
+vi.mock( 'chalk', () => ( {
+	default: {
+		underline: ( text: string ) => text,
+		cyan: ( text: string ) => text
+	}
+} ) );
+vi.mock( 'semver' );
+vi.mock( '@ckeditor/ckeditor5-dev-utils' );
+
+describe( 'provideNewVersionForMonorepository()', () => {
+	const defaultOptions = {
+		packageName: 'test-package',
+		version: '1.0.0',
+		bumpType: 'patch' as ReleaseType,
+		indentLevel: 0
+	};
+
+	beforeEach( () => {
+		vi.mocked( npm.manifest ).mockRejectedValue( new Error( 'Package not found' ) );
+		vi.mocked( npm.checkVersionAvailability ).mockResolvedValue( true );
+		vi.mocked( semver.valid ).mockReturnValue( '1.0.1' );
+		vi.mocked( semver.gt ).mockReturnValue( true );
+		vi.mocked( semver.inc ).mockReturnValue( '1.0.1' );
+		vi.mocked( inquirer.prompt ).mockResolvedValue( { version: '1.0.1' } );
+	} );
+
+	describe( 'Version suggestion and prompt configuration', () => {
+		it.each( [
+			{ bumpType: 'patch', expectedVersion: '1.0.1' },
+			{ bumpType: 'minor', expectedVersion: '1.1.0' },
+			{ bumpType: 'major', expectedVersion: '2.0.0' },
+		] )(
+			'should suggest version for bumpType="$bumpType"',
+			async ( { bumpType, expectedVersion } ) => {
+				const options = {
+					...defaultOptions,
+					bumpType: bumpType as ReleaseType,
+				};
+
+				vi.mocked( semver.inc ).mockReturnValueOnce( expectedVersion );
+				vi.mocked( inquirer.prompt ).mockResolvedValueOnce( { version: expectedVersion } );
+
+				const result = await provideNewVersionForMonorepository( options as any );
+
+				expect( result ).toBe( expectedVersion );
+				expect( semver.inc ).toHaveBeenCalledWith( '1.0.0', bumpType );
+
+				const mockCalls = vi.mocked( inquirer.prompt ).mock.calls as any;
+				const question = mockCalls[ mockCalls.length - 1 ][ 0 ][ 0 ];
+
+				expect( question.default ).toBe( expectedVersion );
+				expect( question.message ).toContain( `suggested: "${ expectedVersion }"` );
+			}
+		);
```

Comment:
```suggestion
		it.each( [
			[ '1.0.1', 'patch' ],
			[ '1.1.0', 'minor' ],
			[ '2.0.0', 'major' ]
		] )(
			'should suggest version "%s" for bumpType "%s"',
			async ( expectedVersion, bumpType ) => {
				const options = {
					...defaultOptions,
					bumpType: bumpType as ReleaseType
				};

				vi.mocked( semver.inc ).mockReturnValueOnce( expectedVersion );
				vi.mocked( inquirer.prompt ).mockResolvedValueOnce( { version: expectedVersion } );

				const result = await provideNewVersionForMonorepository( options as any );

				expect( result ).toBe( expectedVersion );
				expect( semver.inc ).toHaveBeenCalledWith( '1.0.0', bumpType );

				const mockCalls = vi.mocked( inquirer.prompt ).mock.calls as any;
				const question = mockCalls[ mockCalls.length - 1 ][ 0 ][ 0 ];

				expect( question.default ).toBe( expectedVersion );
				expect( question.message ).toContain( `suggested: "${ expectedVersion }"` );
			}
		);
```

This is even better :D